CMakeLists.txt

```diff
@@ -121,8 +121,12 @@ set(OpenGL_GL_PREFERENCE GLVND)
 find_package(Threads REQUIRED)
 find_package(GLUT REQUIRED)
 find_package(OpenGL REQUIRED)
-find_package(PythonInterp REQUIRED)
-find_package(PythonLibs REQUIRED)
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.12")
+    find_package(Python COMPONENTS Interpreter Development)
+else()
+    find_package(PythonInterp REQUIRED)
+    find_package(PythonLibs REQUIRED)
+endif()
```

Comment:
Thanks for making this backwards-compatible. I just noticed that we're already using `cmake_minimum_required(VERSION 3.15.0)` so we already specify a minimum cmake version of 3.15.

That suggests that we can drop the backwards compatibility. Please adjust these changes so that they only support the newer FindPython module.

There is one small tweak we'll need above ~ please use `find_package(Python REQUIRED COMPONENTS ...)` so that cmake will error-out during the configure step when python is not available. It's currently missing the `REQUIRED` option.